### PR TITLE
Regen registry credit class prop

### PR DESCRIPTION
**Pull request by S4mmyb**
Regen Registry Credit Class Creator Proposal 
Author: Sam Bennetts
Co-Authors: Ryan Christofferson, Darren Kong

-- 

Proposal includes overview of registry, our approach to credit design, examples, and plans to decentralize. If you could just give a once over for grammar and suggest any revisions with particular focus on the "credits from other registries" and "plans to decentralize" sections that would be great! Thanks!

### CODE REVIEW

**Comment by S4mmyb** [comment]
Per a conversation I had with @glandua earlier today, who is applying to become a credit class creator? Is it Regen Registry? Or is it the RND staff applying on behalf of what we envision Regen Registry will become: a community governed registry program owned and operated by a experts (scientists, technologists, climate entrepreneurs, project, & methodology developers, land stewards, etc...) committed to ecological regeneration. With this in mind we think it's important to make it clear that it is the RND staff will not own, but rather manage, the multisig address outlined such that it can become decentralized to the community of contributors over time

**Review by ryanchristo — APPROVED** [review]
Added the proposal json and made some adjustments to the introduction, including clarification on the management of the address. Looks good to me. :tada:

File: proposals/2021-12-regen-registry-credit-class-creator/proposal.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "title":"Credit Class Creator - Regen Registry",
+    "description":"This is a parameter change proposal to add a multisig address managed by members of Regen Network Development (RND) to the list of allowed credit class creators. RND staff are applying on behalf of Regen Registry, what is envisioned to become a community governed registry program owned and operated by a experts committed to ecological regeneration.\n\nFor more information, see the long-form proposal: https://github.com/regen-network/governance/tree/main/proposals/2021-12-regen-registry-credit-class-creator",
```
Comment (ryanchristo) [inline proposals/2021-12-regen-registry-credit-class-creator/proposal.json]:
```suggestion
    "description":"This is a parameter change proposal to add a multisig address managed by members of Regen Network Development (RND) to the list of allowed credit class creators. RND staff are applying on behalf of Regen Registry, what is envisioned to become a community governed registry program owned and operated by experts committed to ecological regeneration.\n\nFor more information, see the long-form proposal: https://github.com/regen-network/governance/tree/main/proposals/2021-12-regen-registry-credit-class-creator",
```

**Comment by ryanchristo** [comment]
Made some minor format and grammer revisions and updated the proposal json. Looks good to me. Awaiting final approval from @glandua on content per your conversations.

**Pull request merged**